### PR TITLE
Fix errors in Task. EditBox placeholder to IntlString.

### PR DESCRIPTION
**Pull request by SasLord**

### CODE REVIEW

File: models/task/src/index.ts [hunk 1]
```diff
@@ -67,21 +67,21 @@ export const DOMAIN_TASK = 'task' as Domain
 export const DOMAIN_STATE = 'state' as Domain
 export const DOMAIN_KANBAN = 'kanban' as Domain
 @Model(task.class.State, core.class.Doc, DOMAIN_STATE, [task.interface.DocWithRank])
-@UX('State' as IntlString, task.icon.TaskState, undefined, 'rank')
+@UX(task.string.TaskState, task.icon.TaskState, undefined, 'rank')
 export class TState extends TDoc implements State {
-  @Prop(TypeString(), 'Title' as IntlString)
-  title!: string
+  @Prop(TypeString(), task.string.TaskStateTitle)
+  title!: IntlString
```
Comment (haiodo) [inline models/task/src/index.ts]: Nope, it should not be IntlString, since it customer value.

File: models/task/src/creation.ts [hunk 2]
```diff
@@ -37,15 +38,15 @@ export async function createDeps (client: Client): Promise<void> {
 
 const defaultKanban = {
   states: [
-    { color: 9, title: 'Open' },
-    { color: 10, title: 'In Progress' },
-    { color: 1, title: 'Under review' },
-    { color: 0, title: 'Done' },
-    { color: 11, title: 'Invalid' }
+    { color: 9, title: 'Open' as IntlString },
```
Comment (haiodo) [inline models/task/src/creation.ts]: This is wrong. title is customer value.

File: plugins/task-resources/src/components/KanbanCard.svelte [hunk 3]
```diff
@@ -48,8 +47,9 @@
         <div class="step-lr75"><CommentsPresenter value={object} /></div>
       {/if}
     </div>
-    {#if object.$lookup?.assignee}
-      <Avatar avatar={object.$lookup?.assignee?.avatar} size={'x-small'} />
+    {#if object.$lookup}
```
Comment (haiodo) [inline plugins/task-resources/src/components/KanbanCard.svelte:50]: Why?

**Comment by haiodo** [comment]
@SasLord Please do smaller PRs, per feature (like task/task-model/task-resources)

**Review by haiodo — CHANGES_REQUESTED** [review]
@SasLord Please revert title type changes.

**Review by aplatoff — APPROVED** [review]

**Review by haiodo — APPROVED** [review]

**Pull request merged**